docs/paper/nero.tex

```diff
@@ -484,32 +496,107 @@ \subsubsection{Winternitz Signatures in Bitcoin
     
     pk_size = 20 * k
     ver_size = 2 * d * k
+    sig_size = 21*k
     rec_size = 0
     for i in range(0, n0):
         rec_size += int(i * w)
     
-    total = pk_size + ver_size + rec_size
+    total = pk_size + ver_size + rec_size + sig_size
+    
+    print(f"{d} & {sig_size} & {pk_size} & {ver_size} & {rec_size} & {total} \\\\")
+\end{verbatim}
+\fi
+\begin{table}[H]
+  \centering
+  \begin{tabular}{cccccc}
+    \toprule
+    $d$ & \textbf{Signature} & \textbf{Public Key} & \textbf{Verification Script} & \textbf{Recovery Script} & \textbf{Total} \\
+    \midrule
+    3 & 420 & 400 & 120 & 240 & 1180 \\
+    7 & 294 & 280 & 196 & 165 & 935 \\
+    15 & 210 & 200 & 300 & 112 & 822 \\
+    31 & 189 & 180 & 558 & 105 & 1032 \\
+    63 & 168 & 160 & 1008 & 90 & 1426 \\
+    127 & 147 & 140 & 1778 & 70 & 2135 \\
+    255 & 126 & 120 & 3060 & 48 & 3354 \\
+    \bottomrule
+  \end{tabular}
+  \caption{Different script sizes depending on the $d$ value per each
+    32-bit message. Note that, ``Signature'' column includes the
+    encoding of 32-bit message in it.}\label{tab:winternitz-script-size}
+\end{table}
+
+\subsubsection{Compact Winternitz commitment
+  scheme in Bitcoin}\label{sec:compact-winternitz-in-bitcoin}
+
+As we mentioned in prevoius section, the most contribution to script
+size comes from public key, signature, verification and recovery, but
+do we really need to do that for all of the limbs?
+
+Let's check the checksum of winternitz singature described
+in~\cite{applied-crypto}:
+
+\begin{equation}
+  c \gets d n_0 - (e_1 + \ldots + e_{n_0})
+\end{equation}\label{eq:winternitz-checksum}
```

Comment:
If an equation is small, I prefer writing it inline (this is the case). Also, instead of `\gets`, it is OK to use the `=` sign here (typically, `\gets` is used for describing algorithms where you set/change some variables in the process/in the protocol).

Reply:
I just inherited it from original paper we are referring to, so not sure we need to make any changes here